db/migrate/20200219181808_add_taxa_category_values_to_host_genome.rb

```diff
@@ -0,0 +1,16 @@
+# This adds more values to taxa_category. Initial values were from:
+# https://czi.quip.com/yPgpAPj1dtEC/Hosts-Dropdown-Options
+# See also AddTaxaCategoryToHostGenome.
+class AddTaxaCategoryValuesToHostGenome < ActiveRecord::Migration[5.1]
+  def change
+    unknowns = ['Unknown', 'ERCC only']
+    HostGenome.where(name: unknowns).find_each { |u| u.update(taxa_category: "unknown") }
```

Comment:
It looks like `taxa_category` isn't in the schema.rb. could you add it / any others like `samples_count` ?

Reply:
Sure! I just ignore schema.rb out of habit but I guess any PR that runs a migration should update it.